test/RewriteOriginal/RewriteOriginal.csproj

```diff
@@ -1,7 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
   </PropertyGroup>
+  
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
```

Comment:
I would suggest to add a comment outlining the _intent_ of this command (as post-build targets are not "rare" in .NET, yet, they are not ubiquitous either).